### ISSUE TITLE
Svelte v3 support

### ISSUE THREAD

**Issue by antony** [DeMoorJasper/parcel-plugin-svelte#36]
# 🙋 Feature Request

## Detailed description

Support the upcoming Svelte v3 syntax!

I've done the initial work here: https://github.com/DeMoorJasper/parcel-plugin-svelte/pull/35 but am encountering an issue.

Placing this github feature request here for tracking purposes.

**Comment by DeMoorJasper** [comment]
I think I fixed all the issues in #38

**Comment by DeMoorJasper** [comment]
I've cleaned up this repo a bit and added a working version of your PR here: #38 (I think)
Feel free to review, experiment with it and improve the code :)

**Comment by DeMoorJasper** [comment]
Published as `2.1.0-alpha-0` gonna follow up any bugs/issues with svelte 3 support here. If it is reported to be stable and Svelte 3 stable version is out I'll release 2.1

**Comment by antony** [comment]
Heya - awesome effort, tried this out but I'm encountering a few issues:

https://github.com/antony/svelte-seed/tree/feature/svelte-v3

First one is that `store` is no longer an option to the compiler, so I commented that out in the parcel plugin, and then I got `shared` is not an option, so I commented that out.

Now I'm getting an error that `SvelteComponent` can only be instantiated with `new`. I'm assuming that this is because I commented `shared` out.

I'll have a look and see if I can determine how the rollup plugin manages these things.
